services/app-api/forms/wp.json

```diff
@@ -854,6 +854,104 @@
               }
             ]
           }
+        },
+        {
+          "name": "State & Territory Specific Initiatives: IV. Close-out information (if applicable)",
+          "path": "/wp/state-and-territory-specific-initiatives/close-out-information",
+          "pageType": "entityOverlay",
+          "entityType": "closeOut",
+          "verbiage": {
+            "intro": {
+              "section": "State & Territory Specific Initiatives: IV. Close-out information (if applicable)",
+              "subsection": "Person Centered Planning",
+              "info": [
+                {
+                  "type": "html",
+                  "content":
+                    "Provide initiative description, including target populations and timeframe."
+                }
+              ]
+            },
+            "closeOutWarning": {
```

Comment:
looks like `Close out initiative` & `Return to dashboard` in the alert should be in italics.
We need to add `This action cannot be undone` to the bottom of the alert
The warning icon needs to be updated to a ! in a triangle
<img width="696" alt="Screenshot 2023-09-15 at 4 32 02 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/10237149/a0fde605-0fe2-4f1b-9e7d-d41cbcfb22ff">